tests/Unit/Models/CampaignTest.php

```diff
@@ -61,6 +66,10 @@ public function testToSearchableArray()
             $evergreenWebsiteCampaign->toSearchableArray()['is_evergreen'],
             true,
         );
+        $this->assertEquals(
+            $evergreenWebsiteCampaign->toSearchableArray()['is_group_campaign'],
+            true,
+        );
```

Comment:
This is so nitpicky, but what if we used `assertTrue` and `assertFalse` for these checks? I think it might read a little clearer, especially in scenarios like this one where things get split onto multiple lines.

Reply:
Oooooh heck yes! That is one satisfying refactor https://github.com/DoSomething/rogue/pull/1150/commits/8a43d8ea443919a91bd2c33c6492659b4d93aaaa